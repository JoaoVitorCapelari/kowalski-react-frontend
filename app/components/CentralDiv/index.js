/**
*
* CentralDiv
*
*/

import styled from 'styled-components';

const CentralDiv = styled.div`
  display: flex;
  flex-grow: 1;
  align-items: center;
  justify-content: center;
`;

export default CentralDiv;
