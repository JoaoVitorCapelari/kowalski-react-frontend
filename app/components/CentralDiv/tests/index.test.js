import React from 'react';
import { shallow } from 'enzyme';

import CentralDiv from '../index';

describe('<CentralDiv />', () => {
  it('renders', () => {
    const renderedComponent = shallow(
      <CentralDiv />
    );
    expect(renderedComponent.html()).toMatchSnapshot();
  });
});
