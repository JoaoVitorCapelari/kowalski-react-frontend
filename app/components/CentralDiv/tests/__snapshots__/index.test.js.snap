// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CentralDiv /> renders 1`] = `"<div class=\\"CentralDiv__CentralDiv-jeadek bbknNc\\"></div>"`;
